presto-main/src/main/java/com/facebook/presto/memory/SynchronizedAggregatedMemoryContext.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
I would squash that with the commit that is using this. It would be easier to review this class with the usage context.